File: python/36f_1829A_LoveStory.py
def main():
    ans = []
    check = "codeforces"
    t = int(input())
    for _ in range(t):
        s = input().strip()
        cnt = 0
        for i in range(10):
            if s[i] != check[i]:
                cnt += 1
        ans.append(cnt)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/36f_1829A_LoveStory.cpp
#include<bits/stdc++.h>

int main(){
    int t, cnt;
    std::vector<int> ans;
    std::string s, check = "codeforces";
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>s;
        cnt = 0;
        for(int j = 0; j < 10; j++){
            if (s[j] != check[j]){
                cnt++;
            }
        }
        ans.push_back(cnt);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}